getting-started/scaling-out.md

```diff
@@ -30,72 +29,65 @@ and removed using [`delete_data_node`][delete_data_node].
 
 Note that:
 
-* Data nodes are added as objects locally and connect to an
-  existing PostgreSQL instance.
+* A data node is added as a local object on the access node and
+  contains the configuration needed to connect to a database on a
+  PostgreSQL instance.
 
-* You should already have a running PostgreSQL server on the data node host.
+* You should already have a running PostgreSQL server on an instance
+  that will host a data node database. The data node's database will
+  be created when executing the `add_data_node` command on the access
+  node and should _not_ exist prior to adding the data node.
```

Comment:
It can exist, it will not fail, but it might be good to not encourage it.

```
	/* Create the database with the user as owner. This command might fail
	 * if the database already exists, but we catch the error and continue
	 * with the bootstrapping if it does. */
	res = remote_connection_execf(conn,
								  "CREATE DATABASE %s OWNER %s",
								  quote_identifier(dbname),
								  quote_identifier(username));
```

Reply:
Question is if we should delve into the subtleties of this here. I do say _should_ and not _must_. I figure we'd keep it simple here.